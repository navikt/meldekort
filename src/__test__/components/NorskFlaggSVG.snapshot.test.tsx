import * as React from 'react';
import * as renderer from 'react-test-renderer';
import NorskFlaggSVG from '../../app/components/sprakvelger/NorskFlaggSVG';

it('Sprakvelger', () => {
  const tree = renderer.create(<NorskFlaggSVG />);
  expect(tree).toMatchSnapshot();
});
