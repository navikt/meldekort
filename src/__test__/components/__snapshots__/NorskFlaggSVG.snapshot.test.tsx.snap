// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sprakvelger 1`] = `
<svg
  height={18}
  width={25}
>
  <title>
    Flag_of_Norway
  </title>
  <g
    fill="none"
    fillRule="evenodd"
  >
    <path
      d="M0 0h25v18H0z"
      fill="#EF2B2D"
    />
    <path
      d="M6.875 0h4.688v18H6.875z"
      fill="#FFF"
    />
    <path
      d="M0 6.9h25v4.5H0z"
      fill="#FFF"
    />
    <path
      d="M8.125 0h2.188v18H8.125z"
      fill="#002868"
    />
    <path
      d="M0 7.8h25v2.4H0z"
      fill="#002868"
    />
  </g>
</svg>
`;
